src/apps/loan-list/utils/helpers.ts

```diff
@@ -136,4 +137,40 @@ export const mapFBSRenewedLoanToLoanMetaDataType = (
   });
 };
 
+export const getMaterialInfo = (
```

Comment:
I like the idea of this function. That you can make sure that the object is constructed as you want.
If you could add types for the values it would be an extra benefit.
So you wouldn't have to use the `as Pid` construction.